sn_networking/src/cmd.rs

```diff
@@ -669,7 +617,7 @@ impl SwarmDriver {
                 // be handled.
                 // `self` then handles the request and sends a response back again to itself.
                 if peer == *self.swarm.local_peer_id() {
-                    trace!("Sending query request to self");
```

Comment:
Hmmm, i think the point here with trace was that we don't normally want this info?

These logs are more detailed than we'd want by default. (I believe... but maybe that's not the case).

Reply:
Trace contained some useful info alteast for me, but also contained the stats which were clogging things up.

If certain logs are not useful, we can move them to `trace` down the line, but now if we use `SN_LOG=v`, things are much cleaner to read.